### DISCUSSION TITLE
Add a Gitbook service status page

### DISCUSSION THREAD

**Discussion by ahcsco** [GitbookIO/community#756] [Feature Requests]
It would be handy to have a Gitbook service status page that shows the current status (online or offline or experiencing slowdown) for different services like:

-Gitbook UI 
-Backend Cloud servers' status
-Git sync towards Gitlab and GitHub
-Any third-party integrations

As we've experienced git sync and lag issues in the past, it would be a useful addition.

**Comment by taszap** [comment]
Hi @ahcsco, thanks for your feedback! It already exists under: https://www.gitbookstatus.com

**Reply by ahcsco** [reply]
Perfect. Thanks.
